handlers/features.go

```diff
@@ -537,3 +539,78 @@ func (oh *featureHandler) StoriesSend(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(resp.StatusCode)
 	w.Write(respBody)
 }
+
+func (oh *featureHandler) BriefSend(w http.ResponseWriter, r *http.Request) {
+
+	ctx := r.Context()
+	pubKeyFromAuth, _ := ctx.Value(auth.ContextKey).(string)
+	if pubKeyFromAuth == "" {
+		fmt.Println("no pubkey from auth")
+		w.WriteHeader(http.StatusUnauthorized)
+		return
+	}
+
+	body, err := io.ReadAll(r.Body)
+	r.Body.Close()
+	if err != nil {
+		http.Error(w, "Failed to read requests body", http.StatusBadRequest)
+		return
+	}
+
+	var postData PostData
+	err = json.Unmarshal(body, &postData)
+	if err != nil {
+		fmt.Println("[BriefSend] JSON Unmarshal error:", err)
+		http.Error(w, "Invalid JSON format", http.StatusNotAcceptable)
+		return
+	}
+
+	apiKey := os.Getenv("SWWFKEY")
+	if apiKey == "" {
+		http.Error(w, "API key not set in environment", http.StatusInternalServerError)
+		return
+	}
+
```

Comment:
At this point we should add the `webhook_url` to `postData`. because the `webhook_url` is not coming from the frontend and it's server specific meaning the test and production server have different hostname but the same endpoint, something like 

`"webhook_url": "https://community.sphinx.chat/feature/brief"`
`"webhook_url": "https://people-test.sphinx.chat/feature/brief"`

You can get it like this: `os.Getenv("HOST")`